pricelist_price_base_custom/models/product_pricelist_item.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (C) 2024 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class PricelistItem(models.Model):
+    _inherit = "product.pricelist.item"
+
+    base = fields.Selection(
+        selection_add=[("custom_value", "Custom Value")],
+        ondelete={"custom_value": "set default"},
+    )
+
+    def _compute_base_price(self, product, quantity, uom, date, target_currency):
+        """
+        Override to use custom base price if "custom_value"
+        is selected as rule base and custom_base_price is
+        passed in context
+        """
+        if self.base == "custom_value":
+            return (
+                self.env.context.get("custom_base_price", 0.0)
+                if "custom_base_price" in self.env.context
```

Comment:
@ivs-cetmix Nearly :sweat_smile: 

```suggestion
                return self.env.context.get("custom_base_price", 0.0)
```

is sufficent. `get()` will return `0.0` if `custom_base_price` is not found in context

Reply:
I actually applied suggested changes... but ok, let's try again 😄